### PR TITLE
#2043 Fix to the Visual bug with Trick and two Air Balloons

### PR DESCRIPTION
**Pull request by jrosa**
I am forcing -enditem on target airballoon if the user of Trick does not have an item

### CODE REVIEW

File: data/moves.js [hunk 1]
```diff
@@ -14550,6 +14550,9 @@ exports.BattleMovedex = {
 			if (myItem) {
 				target.setItem(myItem);
 				this.add('-item', target, myItem, '[from] move: Trick');
+			}else {
```
Comment (Slayer95) [inline data/moves.js]: Style

File: data/moves.js [hunk 2]
```diff
@@ -14550,6 +14550,9 @@ exports.BattleMovedex = {
...
+				target.setItem(myItem);
```
Comment (Slayer95) [inline data/moves.js]: Why?

File: data/moves.js [hunk 3]
```diff
@@ -14550,6 +14550,9 @@ exports.BattleMovedex = {
...
+				this.add('-enditem', target, 'Air Balloon');
```
Comment (Slayer95) [inline data/moves.js]: An analogal message should probably be sent if the target pokémon doesn't have an item, as well.

Also, this is missing the source effect parameter, which the client currently doesn't handle properly (it assumes the balloon popped).

Comment (Zarel) [inline data/moves.js]: Well, this line is probably the right idea, except for the whole 'Air Balloon' part, and it should probably be `[silent]` Do we have `[silent]` support for `-enditem`?

**Comment by Zarel** [comment]
Proper fix in fa7263eacdbe8

File: data/moves.js (hunk 1 above)
Comment (Zarel) [inline data/moves.js]: Weird that jscs doesn't catch this.

**Pull request closed without merging**